src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -97,10 +100,16 @@ const NavSearchBar = () => {
     />
   );
 
+  const buildEventDescripiton = (startUtcSeconds: number) => {
+    const weekDay = formatToWeekdayFullname(startUtcSeconds);
+    const time = formatUtcSeconds(startUtcSeconds);
+    return `Event - ${weekDay} at ${time}`;
+  };
```

Comment:
typo is fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/1ae5d879a3e476a6ae049ff7058d7218029c4c1a

Reply:
Since we have substing 'Event - ' here, I don't think that we'll need somewhere exactly such string. 
And for event time we can use `formatRelative`.

Reply:
[NB] Makes sense. Based on how the function looks in the diff you linked above, we could almost get away with it not being a function anymore too (assuming it's only used in that one place) and just inlining it in the prop as:

```typescript
`Event - ${uppercaseFirstChar(formatUtcSecondsRelativeToToday(eventTime))}`;
```

Reply:
done: https://github.com/sparkletown/sparkle/pull/1328/commits/c50dba7f0cdf5c19441f06c1edd9889d40e3cb16